core/backend/src/test/standalone/ViewDefinition.test.ts

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Guid, Id64, Id64String } from "@bentley/bentleyjs-core";
+import { Matrix3d, Range3d, StandardViewIndex, Transform, YawPitchRollAngles } from "@bentley/geometry-core";
+import { Camera, Code, ColorDef, ElementProps, IModel, IModelError, SpatialViewDefinitionProps, SubCategoryAppearance } from "@bentley/imodeljs-common";
+import { assert } from "chai";
+import { AuthorizedBackendRequestContext, CategorySelector, DictionaryModel, DisplayStyle3d, IModelDb, ModelSelector, SpatialCategory, SpatialViewDefinition, StandaloneDb } from "../../imodeljs-backend";
+import { IModelTestUtils, TestUserType } from "../IModelTestUtils";
+
+export async function createNewModelAndCategory(requestContext: AuthorizedBackendRequestContext, rwIModel: IModelDb, parent?: Id64String) {
+  // Create a new physical model.
+  const [, modelId] = await IModelTestUtils.createAndInsertPhysicalPartitionAndModelAsync(requestContext, rwIModel, IModelTestUtils.getUniqueModelCode(rwIModel, "newPhysicalModel"), true, parent);

```

Comment:
use sync function. Get rid of requestContext.

Reply:
Fixed.